docs/DOCUMENTATION.md

```diff
@@ -435,73 +436,78 @@ type Person struct {
 }
 
 func (p Person) String() string {
-	return fmt.Sprintf("%s: %d", p.Name, p.Age)
+	return fmt.Sprintf("[%s/%s] %s: %d", p.InstanceId, p.Id, p.Name, p.Age)
 }
 
+// Example for the multi-instance feature by creating records for different instances
+// and checks that correct instance can access data belonging to it.
 func main() {
-	rand.Seed(time.Now().UnixNano())
-	uId := fmt.Sprintf("P%d", rand.Intn(1_000_0000))
+	uId := "P1337"
 	p1 := &Person{uId, "Bob", 31, "Dev"}
 	p2 := &Person{uId, "John", 36, "Prod"}
 	p3 := &Person{"P3", "Pat", 39, "Dev"}
 
 	SERVICE_ADMIN := "service_admin"
 	SERVICE_AUDITOR := "service_auditor"
 	mdAuthorizer := authorizer.MetadataBasedAuthorizer{}
-	ServiceAdminCtx := mdAuthorizer.GetAuthContext("", SERVICE_ADMIN)
 	instancer := authorizer.SimpleInstancer{}
 
+	ServiceAdminCtx := mdAuthorizer.GetAuthContext("", SERVICE_ADMIN)
 	DevInstanceCtx := instancer.WithInstanceId(ServiceAdminCtx, "Dev")
 	ProdInstanceCtx := instancer.WithInstanceId(ServiceAdminCtx, "Prod")
 
-	ds, _ := datastore.FromEnv(datastore.GetCompLogger(), mdAuthorizer, instancer)
+	// Initializes the Datastore using the metadata authorizer and connection details obtained from the ENV variables.
+	ds, err := datastore.FromEnv(datastore.GetCompLogger(), mdAuthorizer, instancer)
+	if err != nil {
+		log.Fatalf("datastore initialization from env errored: %e", err)
```

Comment:
`%e` is not right. It's used for scientific notation. It's gotta be `%s`

Reply:
oops it was %+v that I wanted